### PR TITLE
Bootstrap requests_queue service

### PR DESCRIPTION
**Pull request by richard-dds**
Set up the `requests_queue` service, as a tornado app, making use of the conventions we've established so far. `scripts/`, `pipenv`, `.ini` config.

Also made a mock `requests/` endpoint with the data I saw in the spreadsheet, though now I'm wondering if that was the right place to put it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+
+# ATST
```
Comment (jordangov) [inline README.md]: Uuuh... looks like we copied a bunch of stuff? I'm guessing we don't want the title, build status, etc from ATST.

File: requests_queue/handlers/request.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
+from json import loads
+import pendulum
+from tornado.web import RequestHandler
+
+
+request = {
```
Comment (jordangov) [inline requests_queue/handlers/request.py]: Sorry, why is this here? I'm unclear why we have stub data in our request handler.

**Review by jordangov — CHANGES_REQUESTED** [review]

File: requests_queue/handlers/request.py (hunk 2 above)
Comment (richard-dds) [reply requests_queue/handlers/request.py]: Removed, though I'm not sure what the issue was with it. Updated the handler to return an empty response.

File: README.md (hunk 1 above)
Comment (richard-dds) [reply README.md]: 👍

**Review by jordangov — APPROVED** [review]

**Comment by bduggan** [comment]
Looks good! 👍

**Review by briandds — APPROVED** [review]
Looks good!

**Pull request merged**